src/object_properties.cpp

```diff
@@ -155,5 +157,9 @@ void ObjectProperties::deSerialize(std::istream &is)
 	infotext = deSerializeString(is);
 	wield_item = deSerializeString(is);
 	can_zoom = readU8(is);
-	glow = readS8(is);
+
+	if (is.good()) {
+		glow = readS8(is);
```

Comment:
why adding glow to the is.good ?

Reply:
It was added yesterday, where this check was forgotten. I'm moving both into a try/catch to ensure there are no errors. Indeed, `is.good` is really not suited for these compatibility checks.